Upgrade Nest to v6 and MiniProfiler to v4

danbrad opened the pull request:
Hi

I have created this pull request to upgrade Nest to v6 and MiniProfiler to v4.

I've run the sample project to show that it works. 

![image](https://user-images.githubusercontent.com/367304/50290078-7d7a8b80-0462-11e9-8f61-d566309f49d4.png)

I have run a build in appveyor https://ci.appveyor.com/project/danbrad/miniprofiler-elasticsearch/builds/21149649

I have also installed the package in my project and it works there too.

Any questions, let me know.

Thanks

danbrad committed:
Upgrade Nest to v6 and MiniProfiler to v4

romansp commented:
Hi, @danbrad and thank you very much for the PR.

Unfortunately, I don't currently use it in any production-ready projects, so can't heavily test that. But I will try to work on that and release to nuget this weekend.

As we are here, I would like to add netstandard support as well. Are you interested in that?

Do you mind if I will add more commits to this PR? I don't know if you enabled [this option](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/).

danbrad commented:
Feel free to add more commits (option is ticked) - I thought about adding netstandard 2 at the time, but decided to get this far first before going any further.

thanks

romansp commented:
Awesome then. I would like to renovate this project and drop support for old versions. Update to new .csproj format and multi-target net461/netstandard2 is definitely on my to-do list.

danbrad commented:
Hmm, looks like MiniProfiler only targets net461 & there is a new package for Asp.Net Core  https://miniprofiler.com/dotnet/AspDotNetCore

Might need to create a MiniProfiler.ElasticSearchCore package instead for that?

romansp commented:
Yes, you're right. I will investigate little more on that question. Will post roadmap to this PR.

danbrad commented:
I've converted the project to the new format - can push up if you like

romansp commented:
Sure, thanks!

danbrad committed:
Convert project to new format

danbrad committed:
Use VS2017 for appveyor builds

danbrad committed:
Add xml documentation back in

romansp committed:
Upgrade packaging, moving to new `dotnet` CLI `pack` command
- Get rid of .nuspec file
- Define package properties in Directory.build.props file
- Added SourceLink support
- Few updates for appveyour nuget packing

romansp commented:
Just merged this. Thank you!

I will continue doing some work in a separate PR. I'm planning to add support for netstandard (will be a separate NuGet package).

romansp commented:
@danbrad, 6.0.0 package is on nuget. Let me know if you hit any problems.

danbrad commented:
Seems to work, thanks!
